### PR TITLE
Fixed the sidebar issue on userside, moved the add user button,fixed …

### PR DESCRIPTION
**Pull request by AbhishekDeb70**
…the add twice isuue ,css issue and routes issue fixed too

### CODE REVIEW

File: client/src/components/Admin/Products/UserInfo.js [hunk 1]
```diff
@@ -30,13 +30,12 @@ const UserInfo = (props) => {
...
         </nav>
-
         <div id="leftCol" className="a-column a-span4 dp_aib_left_column_t1">
           <div class="image-space">
             <h1 class="pdp-title">Hello</h1>
             <h1 class="pdp-name">Check</h1>
-            <img class="rounded-circle z-depth-2" 
-            alt="95x95" src="https://images.unsplash.com/photo-1453728013993-6d66e9c9123a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8dmlld3xlbnwwfHwwfHw%3D&w=1000&q=80"
+            <img class="rounded-circle z-depth-2 ml-4" 
+            alt="95x95" src="https://i.guim.co.uk/img/media/ec27f69b7e4ac14838e8f71842a4cc6db3b8d69c/112_4_1179_708/master/1179.jpg?width=1200&height=1200&quality=85&auto=format&fit=crop&s=1cb08a39c5acdc58423974865c54d7f7"
```
Comment (iMagesh) [inline client/src/components/Admin/Products/UserInfo.js:38]: can you remove this hard coded image link? use bootstrap placeholder instead using the right syntax

**Comment by iMagesh** [comment]
you have conflicts now, resolve that too :)

File: client/package-lock_LOCAL_232123.json [hunk 2]
```diff
@@ -0,0 +1,10810 @@
+{
```
Comment (iMagesh) [inline client/package-lock_LOCAL_232123.json:1]: why did you add these files? these are not necessary

File: client/src/components/Admin/Products/UserInfo.js [hunk 3]
```diff
@@ -30,14 +30,13 @@ const UserInfo = (props) => {
...
-            <h1 class="pdp-title">Hello</h1>
-            <h1 class="pdp-name">Check</h1>
-            <img class="rounded-circle z-depth-2" 
-            alt="95x95" src="https://images.unsplash.com/photo-1453728013993-6d66e9c9123a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8dmlld3xlbnwwfHwwfHw%3D&w=1000&q=80"
-          data-holder-rendered="true"/>
+          <div className="image-space">
+            <h2 className="pdp-title ml-4">{userInfo.first_name}</h2>
+            <h2 className="pdp-name ml-4">{userInfo.last_name}</h2>
+            <img className="rounded-circle z-depth-2 ml-4" 
+            alt="95x95" src="https://cdn.icon-icons.com/icons2/1378/PNG/512/avatardefault_92824.png"
```
Comment (iMagesh) [inline client/src/components/Admin/Products/UserInfo.js:38]: you still haven't used bootstrap?

File: client/.env.example [hunk 4]
```diff
@@ -1,2 +0,0 @@
-REACT_APP_BASE_URL=port
```
Comment (iMagesh) [inline client/.env.example:1]: why did you delete the .env.example?

File: client/.env.production [hunk 5]
```diff
@@ -1 +0,0 @@
-REACT_APP_PROD_BASE_URL=http://ecom-server1.herokuapp.com
```
Comment (iMagesh) [inline client/.env.production:1]: why did you delete this file?

File: client/package-lock_LOCAL_232123.json (hunk 2 above)
Comment (iMagesh) [inline client/package-lock_LOCAL_232123.json:1]: I asked you to delete these kind of files that are generating by git during merge conflicts

**Comment by iMagesh** [comment]
@AbhishekDeb70 respond to my comments?

**Comment by iMagesh** [comment]
@AbhishekDeb70 you have conflicts again :)
can you resolve that again and let me know so we can merge?

File: client/src/App.js [hunk 6]
```diff
@@ -70,7 +59,7 @@ const App = () => {
           path="/admin"
           element={
             <PrivateRoute>
-              <DisplayProducts />
+              <Navigate to='/admin/products' />
```
Comment (iMagesh) [inline client/src/App.js:62]: is this the right thing to do? will it be better to load the component here directly? instead of redirecting and then loading the component?

Comment (AbhishekDeb70) [reply client/src/App.js]: It was display products previously someone changed it not me.

Comment (AbhishekDeb70) [reply client/src/App.js]: Fixed it now

**Comment by AbhishekDeb70** [comment]
> can you resolve that again and let me know so we can merge?

Code is now ready to merge and fixed other issues too.

File: server/routes/admin/users/userInfo.js [hunk 7]
```diff
@@ -4,14 +4,25 @@ const knex = require("../../../utils/dbConfig");
...
+      "address.user_id",
+      "users.id"
+    )
     .select(
       "users.id",
       "users.username",
       "users.email",
       "users.first_name",
       "users.last_name",
       "users.is_admin",
```
Comment (iMagesh) [inline server/routes/admin/users/userInfo.js:18]: use "users.*" instead of writing all the field names
same for address also

Comment (AbhishekDeb70) [reply server/routes/admin/users/userInfo.js:18]: How to do that this was a select query

Comment (iMagesh) [reply server/routes/admin/users/userInfo.js:18]: that you have to figure out

Comment (iMagesh) [reply server/routes/admin/users/userInfo.js:18]: we can reduce a few lines if you do that right? why do we have to write all the field names if we want all the columns from that table?

Comment (AbhishekDeb70) [reply server/routes/admin/users/userInfo.js:18]: We are not selecting all the coloumns

Comment (iMagesh) [reply server/routes/admin/users/userInfo.js:18]: why not? admin side you can display all the columns right?

Comment (iMagesh) [reply server/routes/admin/users/userInfo.js:18]: is there any column that you don't want to show?

Comment (AbhishekDeb70) [reply server/routes/admin/users/userInfo.js]: please check now. although i am surprised to see that i selected from user table on the console log it shows data from address table as well, is it because of the join query that i placed before??

File: server/routes/admin/users/userInfo.js [hunk 8]
```diff
@@ -9,21 +9,7 @@ router.get("/:id", (req, res) => {
...
-      "users.first_name",
-      "users.last_name",
-      "users.is_admin",
-      "users.avatar_url",
-      "address.phone",
-      "address.street",
-      "address.city",
-      "address.state",
-      "address.pin_code",
-      "address.country",
```
Comment (iMagesh) [inline server/routes/admin/users/userInfo.js]: we don't need the address details?

Comment (AbhishekDeb70) [reply server/routes/admin/users/userInfo.js]: This all get passed along when i select the entire values. So the code gets small too and values are passed safely too
